frontend/src/components/core/LockScreen.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { InlineShowIcon } from './Button';
+import ErrorMessage from './Error';
+import { LargeCenterPassword, PrimaryInput } from './Input';
+import Loading from './Loading';
+import { LargeText, SmallHoverText } from './Text';
+
+const Content = styled.div`
+  padding: 0 20%;
+`;
+
+type LockScreenProps = {
+  loading: boolean,
+  error: string,
+  enterPasswordAction: (password: string) => void,
+};
+
+export default function LockScreen(props: LockScreenProps) {
+  // Grab props variables.
+  const {
+    error, loading, enterPasswordAction,
+  } = props;
+
+  const [password, setPassword] = useState('');
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const togglePasswordVisibility = () => {
+    setShowPassword(!showPassword);
+  };
+
+  return (
+    <Content>
+      <LargeText>
+        What is the password?
+      </LargeText>
```

Comment:
I can - the reason it's centered like that is because the display shown is currently the problem page layout, which starts at a higher on the page. However, I could modify how this works (which one of your other comments may recommend, which I'll look into) or just add margin from the top.